src/ray/protobuf/public/events_base_event.proto

```diff
@@ -103,4 +103,7 @@ message RayEvent {
   NodeLifecycleEvent node_lifecycle_event = 15;
   ActorDefinitionEvent actor_definition_event = 16;
   ActorLifecycleEvent actor_lifecycle_event = 17;
+
+  // Unique identifier of the node
+  bytes node_id = 18;
```

Comment:
Since it's a common field, should we set it in a central place: the event agent? I think this can make the code simpler, for example, you don't need to pass node id to GCS. cc @MengjinYan